Jenkinsfile

```diff
@@ -67,14 +70,6 @@ pipeline {
         GIT_COMMITTER_EMAIL = 'mc.stanislaw@gmail.com'
     }
     stages {
-        stage('Kill previous builds') {
-            when {
-                not { branch 'develop' }
-                not { branch 'master' }
-                not { branch 'downstream_tests' }
```

Comment:
I think this actually will cause problems, since we use `downstream_tests` for multiple other pipelines, and this will kill all-but-the-latest of them.

@dylex - you mentioned there was a way to make this more configurable?

Reply:
Looks like you figured it out, but here's a similar example from another project:
```
def keep = !env.BRANCH_NAME.startsWith("PR-")
properties([pipelineTriggers(keep ? [...] : [])])
```